### DISCUSSION TITLE
v.198 is recognized as malware (windows defender)

### DISCUSSION THREAD

**Discussion by Ulvarin** [seerge/g-helper#3327] [General]
v197 is fine to download,
while version 198 is blocked by default :)
I know its false positive but im just wondering what changed between these version so that happened?
![image](https://github.com/user-attachments/assets/f4ed2dbf-e92b-4f25-bce0-3282a90953e5)

**Comment by seerge** [comment]
@Ulvarin hello, not much and nothing that could trigger defender https://github.com/seerge/g-helper/compare/v0.197...v0.198 

False positives can happen. I'm usually submitting a ticket to Microsoft defender to remove that, which I did already :) so hopefully it will be gone soon.

**Reply by seerge** [reply]
@Ulvarin latest update to Defender seem to be OK with 0.198, apparently my submission worked :)
